wandb/sdk/wandb_setup.py

```diff
@@ -215,6 +220,9 @@ def __init__(self, settings=None):
     def __getattr__(self, name):
         return getattr(self._instance, name)
 
+    def set_log_handler(self, handler):
+        self._instance._log_handler = handler
+
```

Comment:
@raubitsj not sure if this is the right way to do this.  I need a reference to close the file on reinit.

Reply:
not sure that is quite right, but it is a step in the right direction.  I think there are more changes needed to do this right since setup() is once per process,  probably want to do the log setup in wandb_init.py and the cleanup in wandb_run.py.  Or better yet move the log setup into an early phase in wand_run.py